cylc/flow/network/schema.py

```diff
@@ -2516,6 +2524,27 @@ class Meta:
     )
 
 
+# TODO: Change to use subscribe arg/default graphql-core has a subscribe field
+# for both Meta and Field, graphene at v3.4.3 does not.. As a workaround
+# the subscribe function is looked up via the following mapping:
```

Comment:
Please create an issue for this TODO and stick the URL into this comment so we can track this.

(note to reviewers, this is not used in cylc-flow, but it is used and extended in cylc-uiserver)

Reply:
> (note to reviewers, this is not used in cylc-flow, but it is used and extended in cylc-uiserver)

Yes, it's here because we intend on having graphql subscriptions on the `cylc-flow` end one day..